man/systemd-detect-virt.xml

```diff
@@ -147,6 +147,11 @@
             <entry><ulink url="https://www.lockheedmartin.com/en-us/products/Hardened-Security-for-Intel-Processors.html">LMHS SRE hypervisor</ulink></entry>
           </row>
 
+          <row>
+            <entry><varname>google</varname></entry>
+            <entry><ulink url="https://cloud.google.com/compute">Google Compute Engine (second generation)</ulink></entry>
```

Comment:
I think we shouldn't claim any generation here. i.e. if a third gen is added we should expose it also as "google" I am sure. Hence just drop that part from the string